Tone/instrument/CymbalSynth.js

```diff
@@ -0,0 +1,304 @@
+
+define(["Tone/core/Tone", "Tone/source/Oscillator", "Tone/instrument/Instrument", 
+	"Tone/component/AmplitudeEnvelope", "Tone/component/Filter", 
+	"Tone/signal/Signal", "Tone/signal/Multiply", "Tone/signal/Scale",
+	"Tone/source/PWMOscillator", "Tone/instrument/SimpleFM"],
+function(Tone){
+
+	"use strict";
+	/**
+	 *  @class  CymbalSynth is composed of 6 Tone.FMSynths, with their 
+	 *  carrier set to a square wave, and the modulator set to a pulse 
+	 *  wave. 
+	 *  Inspiration from [Sound on Sound](http://www.soundonsound.com/sos/jul02/articles/synthsecrets0702.asp).
+	 *
+	 *  @constructor
+	 *  @extends {Tone.Instrument}
+	 *  @param {options} [options] the options availble for the synth
+	 *                             see defaults below
+	 *  @example
+	 * var cymbal = new Tone.CymbalSynth().toMaster();
+	 * cymbal.harmonicity.value = 10;
+	 * cymbal.triggerAttack();
+	 */
+	Tone.CymbalSynth = function(options){
+
+		options = this.defaultArg(options, Tone.CymbalSynth.defaults);
+		Tone.Instrument.call(this, options);
+
+		/**
+		 * The initial strike component of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.strike = new CymbalComponent({
+			"volume" : options.strike.volume,
+			"cutoff" : options.strike.cutoff,
+			"frequencyScalar" : 1.886,
+			"resonance" : options.strike.resonance,
+			"resonanceScalar" : 0.533,
+			"envelope" : {
+				"attack": options.strike.envelope.attack, 
+				"decay": options.strike.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 * The body portion of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.body = new CymbalComponent({
+			"volume" : options.body.volume,
+			"cutoff" : options.body.cutoff,
+			"frequencyScalar" : 1.643,
+			"resonance" : options.body.resonance,
+			"resonanceScalar" : 0.333,
+			"envelope" : {
+				"attack": options.body.envelope.attack, 
+				"decay": options.body.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 *  [harmonicity description]
+		 *  @type {Positive}
+		 *  @signal
+		 */
+		this.harmonicity = new Tone.Signal(options.harmonicity);
+		this.harmonicity.units = Tone.Type.Positive;
+
+		/**
+		 *  The base frequency for the series of oscillators. 
+		 *  All oscillators scale in a harmonic or inharmonic pattern
+		 *  depending on the value of the harmonicity.
+		 *  @type {Frequency}
+		 *  @signal
+		 */
+		this.frequency = new Tone.Signal(options.frequency, Tone.Type.Frequency);
+
+		/**
+		 *  Inharmonic ratio of frequencies based on the Roland TR-808
+		 *  Taken from https://ccrma.stanford.edu/papers/tr-808-cymbal-physically-informed-circuit-bendable-digital-model
+		 *  @private
+		 *  @static
+		 *  @type {Array}
+		 */
+		this.inharmRatios = [1.0, 1.483, 1.932, 2.546, 2.630, 3.897];
```

Comment:
since the `inharmRatio` and `harmRatios` are actually private, static, constant, they should not be put in the constructor function. I would pull them out before the constructor and set them with `var`. Take a look at the way it's done in [Tone.Freeverb](https://github.com/Tonejs/Tone.js/blob/c94861ca5495185206bfb3b03eb957bd44610d00/Tone/effect/Freeverb.js#L21)